.gitignore

```diff
@@ -27,6 +27,7 @@ Icon
 # Files that might appear on external disk
 .Spotlight-V100
 .Trashes
+.venv
```

Comment:
Can you elaborate on what would create this file? Is this just a file that ends up being created in your setup, or is it created by a tool or script from this repository or one of our dependencies?

Reply:
oops. that's something I create to put all tock python venv stuff in. Running make prepush generates errors for .venv folder if I don't add it in .gitignore. 

accidentally committed it. lemme remove it real quick!